### PR TITLE
UUI-582: Add `externalId`

### PR DESCRIPTION
**Pull request by MarcinGie**
[UUI-582](https://schibstedio.atlassian.net/browse/UUI-582) :
- Add `pairId` & `sdrn` properties to `UserProfileResponse`
- Add `Client.getExternalId(pairId:externalParty:suffix:)` method
- Add & fix respective unit tests

### CODE REVIEW

**Review by pklawikowski-schibsted — APPROVED** [review]
LGTM

**Review by bogdan-niculescu-sch — APPROVED** [review]
Changes look good, can we remove the dev setup though?

File: Sources/AccountSDKIOSWeb/Simplified Login/Model/DataModels.swift [hunk 1]
```diff
@@ -40,6 +40,8 @@ extension SimplifiedLoginNamedImageData {
             orderedIconNames = ["dba", "Bilbasen"]
         case .pre:
             orderedIconNames = ["Blocket", "Aftonbladet", "SVD", "Omni", "TvNu"] // Swedish icons as default
+        case .dev:
+            orderedIconNames = ["Blocket", "Aftonbladet", "SVD", "Omni", "TvNu"] // Swedish icons as default
```
[the remark this answers is not in the document]
Comment (MarcinGie) [reply Sources/AccountSDKIOSWeb/Simplified Login/Model/DataModels.swift]: Done 👍

File: Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift [hunk 2]
```diff
@@ -39,6 +40,8 @@ public struct ClientConfiguration {
                 sessionServiceStr = "https://session-service.identity-pre.schibsted.com"
             case .proDk:
                 sessionServiceStr = "https://session-service.login.schibsted.dk"
+            case .dev:
+                sessionServiceStr = "https://session-service.identity-dev.schibsted.com"
```
[the remark this answers is not in the document]
Comment (MarcinGie) [reply Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift]: Done 👍

**Pull request merged**